src/components/Atoms/Logo/Logo.jsx

```diff
@@ -0,0 +1,8 @@
+import { ReactComponent as LogoIcon } from '../../../images/icons/logo-icon.svg';
+import css from './Logo.module.css';
+
+const Logo = () => {
+  return <LogoIcon className={css.LogoIcon} />;
+};
```

Comment:
Can we declare our Logo Component to be more elastic.
I mean that would be nice to pass some props as a height and width.
It will let us use our logo everywhere that we need. 💡

Reply:
Sure I can make it as a separate component but in new branch 😇